lib/ui/event_details/storyline_widget.dart

```diff
@@ -27,29 +27,34 @@ class _StorylineWidgetState extends State<StorylineWidget> {
 
   Widget _buildCaption() {
     var content = <Widget>[
-      new Text(
+      const Text(
         'Storyline',
-        style: new TextStyle(
+        style: const TextStyle(
           fontSize: 16.0,
           fontWeight: FontWeight.w700,
         ),
       ),
     ];
 
     if (_isExpandable) {
-      var action = _isExpanded ? 'collapse' : 'expand';
-
-      content.add(new Padding(
-        padding: const EdgeInsets.only(left: 4.0),
-        child: new Text(
-          '[touch to $action]',
-          style: new TextStyle(
-            fontSize: 12.0,
-            fontWeight: FontWeight.w600,
-            color: Colors.black54,
-          ),
-        ),
-      ));
+      const captionStyle = const TextStyle(
```

Comment:
Now that this got a little longer, I would prefer to extract this into a new method so that it's easier to follow.

Something like:

```dart
  Widget _buildCaption() {
    ...
    if (_isExpandable) {
      content.add(_buildExpandCollapsePrompt());
    }
    ...
  }

  Widget _buildExpandCollapsePrompt() {
    const captionStyle = const TextStyle(
      fontSize: 12.0,
      fontWeight: FontWeight.w600,
      color: Colors.black54,
    );

    if (_isExpanded) {
      return const Padding(
        padding: const EdgeInsets.only(left: 4.0),
        child: const Text('[touch to collapse]', style: captionStyle),
      );
    }

    return const Padding(
      padding: const EdgeInsets.only(left: 4.0),
      child: const Text('[touch to expand]', style: captionStyle),
    );
  }
```